app/models/name/spelling.rb

```diff
@@ -35,7 +35,7 @@ def suggest_alternate_spellings(str)
         results = guess_word("", words.first)
         (2..num).each do |i|
           next unless results.any?
-          next unless (i & 1).zero?
+          next unless i.even?
```

Comment:
@nimmolo : Is my change okay?
RuboCop wanted me to change `(i & 1).zero?` to `i.nobits?(1)`. https://docs.rubocop.org/rubocop/1.74/cops_style.html#stylebitwisepredicate
But both seem like extra mental gymnastics.

Reply:
Thanks for asking. I don't understand what this is doing, though.

Reply:
It's doing a bitwise and of i against the number 1 and then testing if that equals zero.  This will be the case if and only if i is even.  Why it was written the way it is I have no idea.  `git blame` current points @nimmolo , but I'm guessing it was copied from somewhere else.